examples/cart/index.js

```diff
@@ -158,13 +160,37 @@ $(function() {
   ============================================================ */
   function openCart() {
     $('.cart').addClass('js-active');
+    $('.overlay').addClass('js-active');
   }
 
   /* Close Cart
   ============================================================ */
   function closeCart() {
-    $('.cart .btn--close').click(function () {
-      $('.cart').removeClass('js-active');
+    $('.cart').removeClass('js-active');
+    $('.overlay').removeClass('js-active');
+  }
+
+  /* Close Cart Handler
+  ============================================================ */
+  function closeCartHandler() {
+    $('.cart .btn--close').click(closeCart);
+  }
+
+  /* Overlay Handler
+  ============================================================ */
+  function overlayHandler() {
+    $('.overlay').click(closeCart);
+  }
+
+  /* Key Press Handler
+  ============================================================ */
+  function keyPressHandler() {
+    var ESCAPE_KEYCODE = 27
+    $(document).keydown(function(e){
```

Comment:
Also, `evt` is preferred, [per the style guide](https://github.com/Shopify/javascript#2.6) :)